specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/expressRoutePort.json

```diff
@@ -563,6 +563,32 @@
         }
       }
     },
+    "ExpressRouteLinkMacSecConfig": {
```

Comment:
As you are adding new property to the response, is this a breaking change to the existing API?

Reply:
I'm adding new properties. This is not a breaking changes as the API can be called by only one customer. If the customer is using updated SDK then can set and get the new fields. If not, customer can use the API without setting or Getting new fields